rotkehlchen/tests/api/test_misc.py

```diff
@@ -409,3 +409,84 @@ def test_query_all_chain_ids(rotkehlchen_api_server):
     response = requests.get(api_url_for(rotkehlchen_api_server, 'allevmchainsresource'))
     result = assert_proper_response_with_result(response)
     assert result == [{'id': chain.value, 'name': chain.to_name()} for chain in ChainID]
+
+
+def test_cache_deletion(rotkehlchen_api_server):
+    icons_dir = rotkehlchen_api_server.rest_api.rotkehlchen.icon_manager.icons_dir
+    avatars_dir = icons_dir / 'avatars'
```

Comment:
Maybe move `avatars_dir` to `icon_manager`? So that it can be accessed as a property too.

Reply:
hmm okay.

Reply:
But that has nothing to do with icons, so I'm not so sure about that.

Reply:
Yes, has nothing to do with icons indeed, however our avatars logic is already tied to icons logic. So I would ask @LefterisJP about this